src/System.IO.UnmanagedMemoryStream/tests/UmsTests.cs

```diff
@@ -189,5 +189,67 @@ public static void CannotUseStreamAfterDispose()
                 Assert.Throws<ObjectDisposedException>(() => stream.WriteAsync(buffer, 0, buffer.Length).GetAwaiter().GetResult());
             }
         }
+
+        [Fact]
+        public static void CopyToTest()
+        {
+            byte[] testData = Enumerable.Repeat((byte)0x20, 10).ToArray();
+
+            using (var manager = new UmsManager(FileAccess.Read, testData))
+            {
+                UnmanagedMemoryStream ums = manager.Stream;
+                UmsTests.ReadUmsInvariants(ums);
+
+                MemoryStream destination = new MemoryStream();
+                ums.CopyTo(destination);
```

Comment:
Could you add similar tests here for CopyToAsync, verifying that in the case of a destination memory stream the copy completes synchronously?